### PR TITLE
Run pytest for verification

### PR DESCRIPTION
**Pull request by jorisvandenbossche**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -33,6 +33,13 @@ requirements:
 test:
   imports:
     - pygeos
+  source_files:
+    - pygeos/test
```
Comment (jorisvandenbossche) [inline recipe/meta.yaml:37]: The error is `Did not find any source_files for test with pattern pygeos/test`. Does that not work for subdirectories?

(cc @ocefpaf)

Comment (caspervdw) [reply recipe/meta.yaml:37]: The problem here is that pygeos does not package the test directory.

I think this requires a download from GitHub to fix this.

Comment (jorisvandenbossche) [reply recipe/meta.yaml:37]: As far as I understand, that's exactly the goal of `source_files`, to ensure to copy files from the package source, if it's not included in the installed package

Comment (jorisvandenbossche) [reply recipe/meta.yaml:37]: See https://conda-forge.org/docs/maintainer/adding_pkgs.html#copying-test-files

Comment (jorisvandenbossche) [reply recipe/meta.yaml:37]: @ocefpaf do you have any idea here about the source_files ?

Comment (ocefpaf) [reply recipe/meta.yaml:37]: It should work with subdirectories. Checking it now...

Comment (ocefpaf) [reply recipe/meta.yaml:37]: @jorisvandenbossche the tests files are not there. source_files copy from source like @caspervdw mentioned above. We can either use a GitHub tarball or ship the tests with the PyPI sdist. (One could argue that we should always ship tests with in compiled extensions.) Or, the shortest route, copy the test files here. However, that last one is not sustainable in the long run b/c we need to keep them in sync with upstream.

**Comment by ocefpaf** [comment]
@jorisvandenbossche I do not recommend merging with my last commit there, this is just to check how things would work once we have the tests in the tarball.

File: recipe/meta.yaml (hunk 1 above)
Comment (jorisvandenbossche) [reply recipe/meta.yaml]: Ah, OK (will see if I can do a doc clarification PR then). 
I think long term we should maybe include the tests in the sdist then (but not in the wheel)?

Comment (ocefpaf) [reply recipe/meta.yaml]: I would not add it in the wheel.

**Comment by ocefpaf** [comment]
One failure and I believe that is probably b/c we are using latest test against a released version.

```
___________________ ERROR collecting test/test_predicates.py ___________________
test/test_predicates.py:28: in <module>
    pygeos.contains_properly,
E   AttributeError: module 'pygeos' has no attribute 'contains_properly'
```

**Comment by jorisvandenbossche** [comment]
Yes, `contains_properly` is a recent addition in master

**Comment by caspervdw** [comment]
Closing through #42

**Pull request closed without merging**